### PR TITLE
Feat: [Web App] Add tab titles.

### PR DESCRIPTION
**Pull request by Jamerrone**
## What change(s) were made
- Each tab has it's own `h1`/title. Just like the Browser Extension.
- Fix: The `typography.css` file was not being loaded.
- Add a wrapping div to each tab/view (linter fix).

## How to test
- Check any tab/page of the Web App.

## Related Trello ticket(s)
https://trello.com/c/tgbvRBbN

### CODE REVIEW

**Review by bob-voorhoede — APPROVED** [review]
✅

File: projects/web-app/src/views/apple-ios.view.vue [hunk 1]
```diff
@@ -1,92 +1,95 @@
...
-        v-for="(icon, index) in touchIcons"
-        :key="index"
-        :key-name="icon.sizes"
-        :refresh-on="touchIcons"
-      >
-        <template #default>
-          <template v-if="icon.sizes">
-            {{ icon.sizes }}
+  <div>
+    <h1>Apple iOS</h1>
```
Comment (vvandijck) [inline projects/web-app/src/views/apple-ios.view.vue]: Are you sure this needs to be an `<h1>`?

**Review by vvandijck — CHANGES_REQUESTED** [review]
I think a better solution would be to add `title` property to the routes defined in `routes/index.js` and then using those. Maybe in a breadcrumb-ish component that we put below the `<app-header />`.

File: projects/web-app/src/views/apple-ios.view.vue (hunk 1 above)
Comment (Jamerrone) [reply projects/web-app/src/views/apple-ios.view.vue]: Yep, dit is de enige H1 van de website, het wordt ook gebruikt als de page title.

**Comment by vvandijck** [comment]
Looks great! 👍

**Review by vvandijck — APPROVED** [review]

**Pull request merged**